db/seeds.rb

```diff
@@ -555,6 +555,7 @@ def seed_quantity(item_name, organization, storage_location, quantity)
                                       organization: pdx_org,
                                       issued_at: Faker::Date.between(from: 4.days.ago, to: Time.zone.today),
                                       delivery_method: Distribution.delivery_methods.keys.sample,
+                                      created_at: Faker::Date.between(from: 10.days.ago)
                                       comment: 'Urgent')
```

Comment:
Downloaded this to kick the tires -- there is a syntax error here.

Reply:
Sorry for the delay -- I'm still getting an error  from the seed when I download it and run setup...   it looks to me like you need to add a to clause to your between  (see the issue_at two lines above for a model)

rails aborted!
ArgumentError: missing keyword: :to
/Users/clfisher/projects/human-essentials/db/seeds.rb:558:in `block in <top (required)>'
/Users/clfisher/projects/human-essentials/db/seeds.rb:550:in `times'
/Users/clfisher/projects/human-essentials/db/seeds.rb:550:in `each'
/Users/clfisher/projects/human-essentials/db/seeds.rb:550:in `<top (required)>'
Tasks: TOP => db:reset => db:setup => db:seed
(See full trace by running task with --trace)

Reply:
@Josiassejod1 I'm still not positive that github will appropriately notify you if I don't mention you (I'm no github expert)  Please see the comment above this one.   Also, please run the seed to check if there's any other problems before re-submitting.  Thanks!